staging/src/k8s.io/csi-translation-lib/plugins/const.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugins
+
+// LabelMultiZoneDelimiter separates zones for volumes
```

Comment:
* add a comment that this matches the delimiter used by `k8s.io/cloud-provider/volume` and is mirrored here to avoid a large dependency
* make this private to avoid picking up other packages depending on this

Reply:
Excellent suggestion, corrected